admin/install_hl_extractor.sh

```diff
@@ -1,9 +1,17 @@
 #!/bin/sh
 
+if [ $# -eq 1 ]
+  then
+    APP_DIR=$1
+else
+    APP_DIR=$(dirname $0)/..
+    echo "Application directory is not specified. Using default: '$APP_DIR'!"
+fi
+
 # Installing dependencies
-apt-get -y install build-essential libyaml-dev libfftw3-dev libavcodec-dev \
-    libavformat-dev python-dev python-numpy-dev python-numpy git \
-    libsamplerate0-dev libtag1-dev libqt4-dev swig pkg-config
+apt-get -y --force-yes install build-essential libyaml-dev libfftw3-dev \
```

Comment:
ok, fine. If there's a signature error then the process should have included `apt-key add` to load signatures :)